scipy/spatial/ckdtree.pyx

```diff
@@ -459,6 +460,9 @@ cdef class cKDTree:
         is the boxsize along i-th dimension. The input data shall be wrapped
         into :math:`[0, L_i)`. A ValueError is raised if any of the data is
         outside of this bound.
+    build_workers : int, optional
+        Number of workers to use for parallel processing. If <=1 one thread is
+        used. Default: 1.
```

Comment:
In other places, we use `workers` and if set to -1, it adjusts to the machine max number of workers. Same in the other functions.
```suggestion
    workers : int, optional
        Number of workers to use for parallel processing. If -1 is given all
        CPU threads are used. Default is 1.
```

Reply:
This is not the currently implemented behavior, but I can add it to the commit....

Reply:
The -1 you mean? IMHO, I think this would be better if you could. We would have something more coherent with the rest. Thanks!